test/integration/belongsToMany.test.js

```diff
@@ -12,149 +12,162 @@ describe('belongsToMany', function () {
     this.sandbox.restore();
   });
 
-  describe('simple association', function () {
```

Comment:
No code changes, just indentation and wrapping to test for both string and model through